service/src/test/java/org/lfenergy/compas/cim/mapping/mapper/CimToSclMapperTest.java

```diff
@@ -159,231 +159,186 @@ private String readFile() throws IOException {
 
     @Test
     void mapSubstationToTSubstation_WhenCalledWithSubstation_ThenPropertiesMappedToTSubstation() {
-        var cgmesSubstation = mock(CgmesSubstation.class);
-        var expectedId = UUID.randomUUID().toString();
-        var expectedDesc = "Some description";
-
-        when(cgmesSubstation.getId()).thenReturn(expectedId);
-        when(cgmesSubstation.getName()).thenReturn(expectedDesc);
+        var id = UUID.randomUUID().toString();
+        var name = "Some description";
```

Comment:
This still works, but my intuition says it should be `"Some name"`.

Reply:
We can have a discussion about this, because this becomes the description of the Substation in the SCL, but is the name in the CIM CGMES File :-)
But I will change this ;-)